src/Data/Order/LocalStorage/localStorageOrderDataClass.ts

```diff
@@ -630,6 +637,13 @@ async function attributes(): Promise<DataClassSchema> {
       { title: lang === 'de' ? 'Vertriebspartner' : 'Sales partner' },
     ],
     status,
+    technicalAgent: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Technischer Bearbeiter' : 'Techincal agent',
```

Comment:
* [ ] Typo "Technical".

Reply:
Fixed it

Reply:
@weiland-pisa `GET https://web102.crm.pisasales.de/portal/order/schema` responds with:

```json
"technicalAgent": [
  "reference",
  {
    "title": "Techincal agent",
    "to": "User"
  }
],
```